### PR TITLE
CloudCare Synchronous Case Claim

### PR DESCRIPTION
**Pull request by wpride**
screen shot 2016-08-11 at 8 46 07 pm
screen shot 2016-08-11 at 8 45 56 pm

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_controller.js [hunk 1]
```diff
@@ -1,17 +1,34 @@
-/*global FormplayerFrontend */
+/*global FormplayerFrontend, Util */
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_controller.js]: 'Util' is defined but never used (no-unused-vars)

File: corehq/apps/cloudcare/templates/formplayer/query_view.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{% load hq_shared_tags %}
+
+<script type="text/template" id="query-view-list-template">
+    <p><h2><%- title %></h2></p>
+    <table class="table table-hover table-striped table-bordered">
+        <tbody>
+        </tbody>
+    </table>
+    <button class="btn btn-default" type="button" id="query-submit-button">
+        <div>Submit</div>
```
Comment (benrudolph) [inline corehq/apps/cloudcare/templates/formplayer/query_view.html]: nit: don't need this div

File: corehq/apps/cloudcare/templates/formplayer/query_view.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    </button>
+</script>
+
+<script type="text/template" id="query-view-item-template">
+    <td class="col-sm-2">
+        <div><%- text ? text : "" %></div>
```
Comment (benrudolph) [inline corehq/apps/cloudcare/templates/formplayer/query_view.html]: also don't need this div

**Comment by benrudolph** [comment]
👍

**Pull request merged**